app/wama.core/Extensions/UserAccountExtensions.cs

```diff
@@ -47,8 +47,8 @@ public static UserAccountViewModel ToViewModel(this UserAccount user)
 
             switch (user.AccountType)
             {
-                case UserAccountType.Unknown:
```

Comment:
wait you told me to remove unknown since patron account type should not be unknown